packages/ringcentral-integration/modules/ConferenceCall/index.js

```diff
@@ -502,6 +502,10 @@ export default class ConferenceCall extends RcModule {
     return null;
   }
 
+  getOnlinePartyIdStr(id) {
```

Comment:
We can remove this code now, since we have shouldComponentUpdate within the CallAvatar.js